samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -100,7 +101,9 @@ static const char link_usage_str[] =
 	"                Has impact after the bootup\n"
 	"  edrx:         Enable/disable eDRX with default or with custom parameters\n"
 	"  psm:          Enable/disable Power Saving Mode (PSM) with\n"
-	"                default or with custom parameters\n";
+	"                default or with custom parameters\n"
+	"  rai:          Enable/disable RAI feature. Actual use must be set for sockets\n"
+	"                with 'sock rai' command. Effective when going to normal mode.\n";
```

Comment:
If you want to say about the actual usage, then ping should be also mentioned in addition to mentioned "sock rai". IMO, you could just say more generically about the actual usage.

Reply:
I'll write generally:
Actual use must be set for commands supporting RAI